### ISSUE TITLE
Merged cols and rows

### ISSUE THREAD

**Issue by geoirb** [qax-os/excelize#1018]
**Description**

In my task i needed know how many rows and cols merge with cell and i wrote function for get this information

I propose to add this function to the main repository
```golang
// GetMergeCells provides a function to get the number of merged rows and columns by axis cell
// from a worksheet currently.
func (f *File) GetNumMergeCell(sheet string, axis string) (colNum int, rowNum int, err error) {
	ws, err := f.workSheetReader(sheet)
	if err != nil {
		return
	}

	if ws.MergeCells != nil {
		for i := range ws.MergeCells.Cells {
			ref := ws.MergeCells.Cells[i].Ref
			cells := strings.Split(ref, ":")
			if cells[0] == axis {
				col1, row1, _ := CellNameToCoordinates(cells[0])
				col2, row2, _ := CellNameToCoordinates(cells[1])
				colNum, rowNum = col2-col1+1, row2-row1+1
				return
			}
		}
	}
	colNum = 1
	rowNum = 1
	return
}
```

I wrote test for this function

**Comment by xuri** [comment]
Thanks for your issue, following the minimum usable design principle, I don't suggest adding a new API into the base library if developers could be easy to implement any feature by the wrapper or composite the existing APIs. You can get the merged cells to range by [`GetMergeCells`](https://pkg.go.dev/github.com/xuri/excelize/v2#File.GetMergeCells) and parse the column and rows base on it.

**Comment by geoirb** [comment]
@xuri  algorithm complexity my function is O(n), algorithm proposed method is O(n^2),  and in the lib code i saw a violation the minimum usable

**Comment by xuri** [comment]
The issues #1382 and #1448 was similar with this, the performance of the `GetMergeCells` function has been improvement by #1500, and this optimization has been released in the version 2.7.1. Please upgrade to the latest version. I've closed this issue, if you have any question, please let me know, and you can reopen this anytime.
